src/_Color.sass

```diff
@@ -7,3 +7,4 @@ $color-primary: #9b4dca !default
 $color-secondary: #606c76 !default
 $color-tertiary: #f4f5f6 !default
 $color-quaternary: #d1d1d1 !default
+$color-quinary: #545b61 !default
```

Comment:
it is useless to set this variable, you should use something like this in _Button.sass file:
````sass
background-color: darken($color-secondary, 20%)
border-color: darken($color-secondary, 20%)
````

20% has to be adjusted to match your color-quinary